--- openbabel-2.3.2/scripts/CMakeLists.txt~	2012-10-03 21:08:52.000000000 +0200
+++ openbabel-2.3.2/scripts/CMakeLists.txt	2014-01-18 18:02:07.812085094 +0100
@@ -222,14 +222,6 @@
       endif (NOT WIN32)
     endif (RUN_SWIG)
 
-    add_library(bindings_perl MODULE ${openbabel_SOURCE_DIR}/scripts/perl/openbabel-perl.cpp)
-    target_link_libraries(bindings_perl ${PERL_LIBRARY} ${BABEL_LIBRARY} ${PERL_EXTRA_LINK_FLAGS})
-    set_target_properties(bindings_perl PROPERTIES
-        PREFIX ""
-        OUTPUT_NAME OpenBabel)
-    add_dependencies(bindings_perl openbabel)
-    install(TARGETS bindings_perl LIBRARY DESTINATION ${LIB_INSTALL_DIR})
-    install(FILES ${openbabel_SOURCE_DIR}/scripts/perl/OpenBabel.pm DESTINATION ${LIB_INSTALL_DIR})
 
     set(PERL_INSTALLDIRS_TEXT "")
     if (OBPERL_INSTALLDIRS)
