--- openbabel-2.3.0/scripts/ruby/extconf.rb.orig	2010-10-23 03:54:40.000000000 +0200
+++ openbabel-2.3.0/scripts/ruby/extconf.rb	2011-01-15 10:45:27.430364132 +0100
@@ -5,8 +5,10 @@
 require 'rbconfig'
 if Config::CONFIG["arch"] =~ /universal-darwin/
   case `uname -smr`.chomp
-  when "i386" : ENV['ARCHFLAGS'] = '-arch i386'
-  when "ppc"  : ENV['ARCHFLAGS'] = '-arch ppc'
+  when "i386"
+	ENV['ARCHFLAGS'] = '-arch i386'
+  when "ppc"
+	ENV['ARCHFLAGS'] = '-arch ppc'
   end
 end
 
