--- scripts/CMakeLists.txt.orig	2012-10-04 05:08:52.000000000 +1000
+++ scripts/CMakeLists.txt	2012-10-15 15:48:42.051537351 +1000
@@ -91,7 +91,7 @@
             PREFIX ""
             SUFFIX .so )
         add_dependencies(bindings_python openbabel)
-        install(TARGETS bindings_python LIBRARY DESTINATION ${LIB_INSTALL_DIR})
+        install(CODE "execute_process(COMMAND ${PYTHON_EXECUTABLE} ${openbabel_BINARY_DIR}/scripts/python/setup.py install --root=\$ENV{DESTDIR}/ WORKING_DIRECTORY ${openbabel_BINARY_DIR}/scripts)" LIBRARY DESTINATION ${LIB_INSTALL_DIR})
         install(FILES ${openbabel_SOURCE_DIR}/scripts/python/openbabel.py DESTINATION ${LIB_INSTALL_DIR})
         install(FILES ${openbabel_SOURCE_DIR}/scripts/python/pybel.py DESTINATION ${LIB_INSTALL_DIR})
     else(NOT WIN32)
--- scripts/python/setup.py.orig	2011-09-16 04:45:02.000000000 +1000
+++ scripts/python/setup.py	2012-10-15 15:24:30.659979953 +1000
@@ -16,6 +16,7 @@
 obExtension = Extension('_openbabel',
                  [os.path.join(srcdir, "openbabel-python.cpp")],
                  include_dirs=[os.path.join(srcdir, "..", "..", "include"),
+                               os.path.join("..", "..", "include"),
                                os.path.join("..", "include")],
                  library_dirs=[os.path.join(srcdir, "..", "..", "lib"),
                                os.path.join(srcdir, "..", "..", "lib64"),
@@ -24,7 +24,7 @@
                  )
 
 if "build" in sys.argv:
-    shutil.copyfile(os.path.join(srcdir, "pybel_py%dx.py" % sys.version_info[0]), "pybel.py")
+    shutil.copyfile(os.path.join(srcdir, "pybel.py"), "pybel.py")
     shutil.copyfile(os.path.join(srcdir, "openbabel.py"), "openbabel.py")
 
 setup(name='openbabel',
